src/screens/BookingsSummary.tsx

```diff
@@ -0,0 +1,181 @@
+    // fetch bookings from backend
+    try {
+      const response = await fetch(`${URL}/bookings`, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      })
+      const data = await response.json()
+      if (!response.ok) {
+        setError(data.message)
+        return
+      }
+      setData(data)
+    } catch (error) {
+      setError('Something went wrong, please try again later')
+    }
+  }
+
+  const renderBookings = () => {
+    if (data) {
+      return data.map((booking) => {
+        return (
+          <Row>
+            <RowBooking
+              key={booking.id}
+              id={booking.id}
+              locationName={booking.locationName}
+              locationId={booking.locationId}
+              price={booking.price}
+              period={booking.period}
+              status={booking.status}
+            />
+            <TicketBooking
+              key={booking.id}
+              id={booking.id}
+              locationName={booking.locationName}
+              locationId={booking.locationId}
+              price={booking.price}
+              period={booking.period}
+              status={booking.status}
+            />
+          </Row>
+        )
+      })
+    }
+  }
+  console.log(data)
+
+  return (
+    <Container fluid className="booking-summary fill bg-dark">
+      <Row className="">
+        <Col md="4">
+          <h2 className="text-white page-title fw-bold">Your bookings</h2>
+        </Col>
+      </Row>
+      <Row className="booking-panel mt-4 mx-auto">
+        <div className="d-none d-lg-block">
+          <BookingsSummaryHeader />
+        </div>
+        {renderBookings()}
+      </Row>
+    </Container>
+  )
+}
+
+const BookingsSummaryHeader: React.FC = () => {
+  return (
+    <Container>
+      <Row className="p-3 ms-1 flex-row align-items-center booking-summary-header justify-content-between ">
+        <Col md="3" className='p-0'>
+          <h5 className="fw-semiBold">location</h5>
+        </Col>
+        <Col md="1" className="d-xl-none w-auto p-0">
+          <h5 className="fw-semiBold">period</h5>
+        </Col>
+        <Col md="3" className="d-none d-xl-block p-0 text-center">
+          <h5 className="fw-semiBold">period</h5>
+        </Col>
+        <Col md="2" className="d-xl-none text-center p-0">
+          <h5 className="fw-semiBold">id</h5>
+        </Col>
+        <Col md="2" lg="1" className="d-none d-xl-block id-header w-auto">
+          <h5 className="fw-semiBold">bookingId</h5>
+        </Col>
+        <Col md="2" lg="1" className="text-center">
+          <h5 className="fw-semiBold">price</h5>
+        </Col>
+        <Col md="1" className="status-header text-center w-auto">
+          <h5 className="fw-semiBold">status</h5>
+        </Col>
+        <Col md="2" xl="1" className="text-center">
+          <h5 className="fw-semiBold">action</h5>
+        </Col>
+      </Row>
+    </Container>
+  )
+}
```

Comment:
BookingSummary.tsx 148 - 179
![image](https://user-images.githubusercontent.com/88868804/224487227-c0539887-b54f-4d2e-a7a4-5dc44b70a790.png)
- This code feels sick. I think it's better if you can use an array.map method to this code section and use the same style(sass or bootstrap) and use the key in map to directly change styles for d-none